repair_tests/repair_test.py

```diff
@@ -1195,6 +1195,43 @@ def run_repair():
         else:
             node1.nodetool('repair keyspace1 standard1 -inc -par')
 
+    @since('3.0')
+    def test_repair_one_node_cluster(self):
+        self._repair_abort_test(nodes=1, rf=2)
+
+    @since('3.0')
+    def test_repair_one_node_in_local_dc(self):
+        options = ['--in-local-dc']
+        support_iuk = self.cluster.version() >= "3.11.10"
+        if support_iuk:
+            options = ["--ignore-unreplicated-keyspaces"] + options
+        self._repair_abort_test(options=options, nodes=[1, 1], rf={'dc1': 1, 'dc2': 1}, no_common_range=True)
+
+    def _repair_abort_test(self, options=[], nodes=1, rf=1, no_common_range=False):
+        cluster = self.cluster
+        logger.debug("Starting cluster..")
+        cluster.populate(nodes).start(wait_for_binary_proto=True)
+        
+        node1 = self.cluster.nodelist()[0]
+        session = self.patient_cql_connection(node1)
+        create_ks(session, 'ks', rf=rf)
+
+        support_preview = self.cluster.version() >= "4.0"
+        if support_preview:
+            logger.debug("Preview repair")
+            out = node1.repair(["--preview"] + options)
+            if no_common_range:
+                assert "Nothing to repair for " in str(out), "Expect 'Nothing to repair for '"
+
+        logger.debug("Full repair")
+        node1.repair(["--full"] + options)
+
+        logger.debug("Incremental repair")
```

Comment:
I think `options` should be passed to both full and normal repair.
The only reason for this `if` was IMO `--inc` flag.
And lack of options in the second call was a bug.
So I would just keep:
```
node1.repair(options)
```
insted of the whole `if` statement.

Reply:
> I think `options` should be passed to both full and normal repair.
> The only reason for this `if` was IMO `--inc` flag.

In such case `else` would contain `node1.repair(options)` instead of `node1.repair()`. So it feels to me that the passed options (`--in-local-dc`) were the reason, however I cannot check this in C* documentation, since it don't find for such old version of C*.

Reply:
there is no need to have `else` at all.

Reply:
I tested locally and it works on any >=3.0 version.